Foundation/JSONEncoder.swift

```diff
@@ -34,7 +34,7 @@ open class JSONEncoder {
         public static let prettyPrinted = OutputFormatting(rawValue: 1 << 0)
 
         /// Produce JSON with dictionary keys sorted in lexicographic order.
-        @available(macOS 10.13, iOS 11.0, watchOS 4.0, tvOS 11.0, *)
+        @available(OSX 10.13, iOS 11.0, watchOS 4.0, tvOS 11.0, *)
```

Comment:
I think this reverts some earlier changes made to normalize the availability macros.

Reply:
Yes @saiHemak this needs to match the version in the overlay: https://github.com/apple/swift/blob/master/stdlib/public/SDK/Foundation/JSONEncoder.swift#L35